### ISSUE TITLE
Exception thrown while opening the wallet app

### ISSUE THREAD

**Issue by bensam282** [mycelium-com/wallet-android#441]
I can't open the mycelium app anymore, it crashes as soon as the app is launched with an exception:

```
12-22 21:30:50.670  7096  7096 E AndroidRuntime: FATAL EXCEPTION: main
12-22 21:30:50.670  7096  7096 E AndroidRuntime: Process: com.mycelium.wallet, PID: 7096
12-22 21:30:50.670  7096  7096 E AndroidRuntime: java.lang.RuntimeException: Unable to create application com.mycelium.wallet.WalletApplication: java.lang.NullPointerException: Attempt to read from field 'com.mrd.bitlib.model.TransactionOutput[] com.mrd.bitlib.model.Transaction.outputs' on a null object reference
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6037)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at android.app.ActivityThread.-wrap1(Unknown Source:0)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1756)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:105)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:164)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6809)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to read from field 'com.mrd.bitlib.model.TransactionOutput[] com.mrd.bitlib.model.Transaction.outputs' on a null object reference
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at com.mycelium.wapi.wallet.AbstractAccount.isColuTransaction(AbstractAccount.java:864)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at com.mycelium.wapi.wallet.AbstractAccount.isColuDustOutput(AbstractAccount.java:876)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at com.mycelium.wapi.wallet.AbstractAccount.calculateLocalBalance(AbstractAccount.java:482)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at com.mycelium.wapi.wallet.bip44.Bip44Account.<init>(Bip44Account.java:68)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at com.mycelium.wapi.wallet.WalletManager.loadBip44Accounts(WalletManager.java:590)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at com.mycelium.wapi.wallet.WalletManager.loadAccounts(WalletManager.java:570)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at com.mycelium.wapi.wallet.WalletManager.<init>(WalletManager.java:98)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at com.mycelium.wallet.MbwManager.<init>(MbwManager.java:9618)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at com.mycelium.wallet.MbwManager.getInstance(MbwManager.java:172)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at com.mycelium.wallet.WalletApplication.onCreate(WalletApplication.java:47)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1129)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6034)
12-22 21:30:50.670  7096  7096 E AndroidRuntime: 	... 8 more

```

Restarting the device does not help, I haven't tried clearing data yet
My Device: OnePlus 3 A3000 (build number 171116)
Android 8.0.0 (Oxygen OS 5.0)

Mycelium Wallet app version: 2.9.10.6

**Comment by Giszmo** [comment]
Thank you for reporting this with a stacktrace even. I had looked into this earlier and don't see a quick fix and neither the root of what is triggering it, as we have very few people affected. We have to fix it and will probably get to work on it this week.

**Comment by Giszmo** [comment]
Hi @bensam282,

could you please let me know if [this apk](https://drive.google.com/file/d/1n0FxlKIuxBJKIBEW65D_mYq9ZV7JU2zo/view?usp=sharing) works for you? It is signed with the release key, with the version incremented, so it should install on top of the playstore version as an update without issues.

**Comment by Giszmo** [comment]
Another user who contacted us via support@mycelium.com tried the apk and it worked for him, so more review, we will release a fix shortly. Tomorrow probably.

**Comment by Giszmo** [comment]
fixed in 2.910.7
